src/protocols/http.hpp

```diff
@@ -558,6 +558,9 @@ class Request : public NonCopyable, public NonMovable {
         // Extend settings with address and port to connect to
         settings["port"] = serializer.port;
         settings["address"] = serializer.address;
+        if (serializer.schema == "httpo") {
+            settings["socks5_proxy"] = "127.0.0.1:9050";  // over Tor
+        }
```

Comment:
This should not override the proxy settings that are specified by the user. That is it should be possible to specify an alternate Tor SOCKS port and make sure such setting has priority over defaulting to 9050.